app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
The variable name `command_list` is not very descriptive. Consider using more descriptive names, such as `command_parts` or `command_elements`, to improve code readability.